### DISCUSSION TITLE
Printer pausing on layer changes

### DISCUSSION THREAD

**Discussion by asincity3** [OrcaSlicer/OrcaSlicer#6220] [Q&A]
I have a Tevo Little Monster with an upgraded 1.2 Big Tree Tech board. It has a volcano .6 nozzle and I am getting a bunch of blobs and looking like burnt plastic because the nozzle stops for several seconds in between layers. None of my other printers do this. I am using the current version of Orca slicer. And tried many setting changes. Any ideas?

**Comment by lin-ycv** [comment]
Sound like power lose protection. 
When you do layer changes the printer write the current layer to the SD card, so if it experience power loss, it knows where to resume from. 
But if your board/SD card is slow to write, the printer will pause at each layer changes.
